package.json

```diff
@@ -102,7 +102,8 @@
     "format:tslint": "tslint --project tsconfig.json --fix",
     "format": "yarn format:prettier && yarn format:tslint",
     "test": "TZ=UTC jest",
-    "updateSnaps": "TZ=UTC jest -u"
+    "updateSnaps": "TZ=UTC jest -u",
+    "version": "yarn lerna version --no-git-tag-version --no-push --no-changelog"
```

Comment:
Not necessary for this release but we could probably relocate this to `lerna.json` and delete `command.publish`

Reply:
Just found out that `version` script here will run twice since `lerna` runs `yarn version` as a lifecycle. I'll make the changes in `lerna.json` for the above comment and rename this script to something else.